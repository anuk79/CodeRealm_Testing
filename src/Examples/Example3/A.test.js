import React from 'react';
import { shallow, mount } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import A from './A';

describe.skip('A component', () => {
  it('should SHALLOW render correctly', () => {
    const wrapper = shallow(<A />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
  it('should MOUNT correctly', () => {
    const wrapper = mount(<A />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
    wrapper.unmount();
  });
});