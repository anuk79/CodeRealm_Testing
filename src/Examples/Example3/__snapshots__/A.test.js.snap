// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A component should MOUNT correctly 1`] = `
<A>
  <B>
    <C>
      <div>
        This is component "C"
      </div>
    </C>
  </B>
</A>
`;

exports[`A component should SHALLOW render correctly 1`] = `
<B>
  This is component "A"
</B>
`;
